zulip_bots/zulip_bots/bots/followup/test_followup.py

```diff
@@ -26,4 +27,6 @@ def test_bot(self):
               'subject': 'foo_sender@zulip.com',
               'content': 'from foo_sender@zulip.com: I have completed my task'}),
         ]
-        self.check_expected_responses(expected_send_message, expected_method='send_message')
+        with self.mock_config_info(self.mock_config):
+            self.initialize_bot()
+            self.check_expected_responses(expected_send_message, expected_method='send_message')
```

Comment:
Can you add a test case where we have a different stream than default? That way we can be certain that the code works as expected. For example, right now, the code wasn't working but the test was passing because the testcase was only testing the default stream name.